packages/terra-slide-panel/src/SlidePanel.jsx

```diff
@@ -53,6 +58,7 @@ const propTypes = {
 };
 
 const defaultProps = {
+  ariaLabel: 'Details Panel',
```

Comment:
Do we require translations for the default value of `aria-label`?

Reply:
I don't think it should have a default prop value -- as the generic slide-panel containter can be used for a wide variety of uses well beyond the common Details Panel, include some layouts using the child panel as the "primary"/navigation panel content that launches pre-opened with the `panelBehavior=squish` prop value. 

I would prefer to leave the new `ariaLabel` prop with no default value - force consumers to add it if their situation needs it for accessibity and label it correctly for their usage.